assembl/static2/js/app/constants.js

```diff
@@ -106,6 +106,14 @@ export const MESSAGE_VIEW = {
   brightMirror: 'brightMirror'
 };
 
+export const defaultPasswordRequirements = [
```

Comment:
Okay, fair enough. We'll create an endpoint to fetch the specific passwords configured on the page.